Automattic/jetpack
# Social | Update connections modal UI

## Proposed changes:

* Update the connections modal UI as per the design

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






- Ensure that feature flag is ON - `define( 'JETPACK_SOCIAL_USE_ADMIN_UI_V1', true );`
- Test the following for both Jetpack and Social plugin
    - Goto settings page
    - Open "Add new connection" modal
    - Confirm that the UI looks as per the design - p1HpG7-swB-p2
    - Add some connections
    - Confirm that you see "Connected" in green for active connection services
    - Add more than one connection to a service
    - Confirm that you see "N connections"

<img width="1048" alt="Screenshot 2024-05-16 at 5 56 06 PM" src="https://github.com/Automattic/jetpack/assets/18226415/d2f5bf97-b69d-4753-990a-86f1cb9d9cb5">
<img width="1051" alt="Screenshot 2024-05-16 at 5 56 14 PM" src="https://github.com/Automattic/jetpack/assets/18226415/64b2eacd-bde2-4f98-98c3-e949c292f661">
<img width="1057" alt="Screenshot 2024-05-16 at 5 56 52 PM" src="https://github.com/Automattic/jetpack/assets/18226415/69b0d4bc-10f7-4759-8034-962eb0ddf1ce">
<img width="1053" alt="Screenshot 2024-05-16 at 5 57 06 PM" src="https://github.com/Automattic/jetpack/assets/18226415/11373749-77bd-4d74-949e-ba02e96b47c1">
